.env.example

```
@@ -1,3 +1,4 @@
 STRIPE_SECRET=...
 PRIVATE_KEY=...
-SENDGRID_API_KEY=...
\ No newline at end of file
+SENDGRID_API_KEY=...
```

Let's prefix with the prefix of sendgrid `SG.` if I remember but that's to confirm
```suggestion
SENDGRID_API_KEY=SG....
```